ironfish/src/account/accounts.ts

```diff
@@ -1122,8 +1128,10 @@ export class Accounts {
           const header = await this.chain.getHeader(Buffer.from(blockHash, 'hex'))
           Assert.isNotNull(header)
           const main = await this.chain.isHeadChain(header)
-          if (main) {
+          if (main && header.sequence + minimumBlockConfirmations < heaviestHead.sequence) {
```

Comment:
I think there's an off-by-one compared to how confirmations are calculated elsewhere. I'll fix it up quick since this has been sitting around for a while!